### PR TITLE
added schemas, endpoints and testdata

### PR DESCRIPTION
**Pull request by smisidjan**

### CODE REVIEW

File: Installation/Schema/pdd.GeografischePositie.schema.json [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+  "version": "0.0.1",
+  "type": "object",
+  "description": "",
+  "required": ["Longitude", "Lattitude"],
+  "properties": {
+    "Longitude": {
+      "type": "string",
+      "required": true
+    },
+    "Lattitude": {
```
Comment (rjzondervan) [inline Installation/Schema/pdd.GeografischePositie.schema.json:14]: Are you sure about the spelling of 'lattitude', as far as I can remember (and find), it should be 'latitude'

**Review by rjzondervan — APPROVED** [review]
Seems fine, one small question about spelling, but that can be an inheritance from the original API.

File: Installation/Schema/pdd.GeografischePositie.schema.json (hunk 1 above)
Comment (smisidjan) [reply Installation/Schema/pdd.GeografischePositie.schema.json:14]: Should we just rename it? So it is spelled correctly

Comment (rjzondervan) [reply Installation/Schema/pdd.GeografischePositie.schema.json:14]: No, if it is like this in the original API it's fine, let's not make our lives difficult by trying to correct others.

Comment (rjzondervan) [reply Installation/Schema/pdd.GeografischePositie.schema.json:14]: If it doesn't make your life more difficult, that seems like a good idea.

**Review by rjzondervan — APPROVED** [review]

**Pull request merged**